contracts/JBPaymentTerminalStore.sol

```diff
@@ -163,16 +169,17 @@ abstract contract JBPaymentTerminalStore is IJBPaymentTerminalStore {
     The current total overflow is represented as a fixed point number with 18 decimals.
 
     @param _projectId The ID of the project to get total overflow for.
+    @param _decimals The number of decimals that the fixed point overflow should include.
     @param _currency The currency that the total overflow should be in terms of.
 
     @return The current total amount of overflow that project has across all terminals.
   */
-  function currentTotalOverflowOf(uint256 _projectId, uint256 _currency)
-    external
-    view
-    returns (uint256)
-  {
-    return _currentTotalOverflowOf(_projectId, _currency);
+  function currentTotalOverflowOf(
+    uint256 _projectId,
+    uint256 _decimals,
```

Comment:
Shouldn't we, for frontend's sake, store decimal[currency] in JBCurrency?

Reply:
Not sure what you mean.

Reply:
Not sure what I meant, discard ':)